src/cards/cards.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import sequelize from 'sequelize';
+import { Item } from 'src/items/items.model';
+import { Card } from './cards.model';
+import { CreateCardDto } from './dto/create-card.dto';
+
+interface CardResult {
+  id: number;
+  product_id: number;
+  item_id: number;
+  price: number;
+  name: string;
+  category: string;
+  quantity: number;
+  sum: number;
+}
+
+@Injectable()
+export class CardsService {
+  static createCard: any;
+  constructor(@InjectModel(Card) private cardRepository: typeof Card) {}
+
+  async createCard(dto: CreateCardDto): Promise<Card> {
+    const card = await this.cardRepository.create(dto);
+    return card;
+  }
+
+  async getCardById(id: string) {
+    const userId = Number(id);
+    const card: Array<CardResult> = await this.cardRepository.sequelize.query(
+      `
```

Comment:
Raw SQL query is not easy to read. Please change it to native Sequelize [joins](https://sequelize.org/docs/v6/advanced-association-concepts/eager-loading/#:~:text=This%20is%20the%20case%20because,at%20least%20one%20matching%20children.)